hasher-matcher-actioner/hmalib/lambdas/local_bucket_hasher.py

```diff
@@ -0,0 +1,44 @@
+import os
+import json
+import boto3
+
+from hmalib.common.logging import get_logger
+
+logger = get_logger(__name__)
+sqs_client = boto3.client("sqs")
+
+PDQ_IMAGES_QUEUE_URL = os.environ["PDQ_IMAGES_QUEUE_URL"]
+
+
+def lambda_handler(event, context):
+    if is_s3_event(event):
+        event["local_bucket"] = True
+        sqs_client.send_message(
```

Comment:
This can be made more generic. 

Instead of sending to the PDQ IMAGES QUEUE, if you make an API call to the submissions API, that would allow changes to the submissions API to automatically benefit this.

If we start supporting another image hashing algorithm, this will need to be reworked in its present state. However, if you send the event to the submissions API, there is a single point of maintenance.

Submission API needs the following attributes:
`content_id`: Which in this case will be the full s3_url
`content_type`: Which will be photo to begin with, but can be expanded to video easily.
`url`: This could be a pre-signed S3 URL. You can generate one from the bucket + key combo.

This removes the need for having custom events or code to handle new use-cases.

If we go as close to the main-line as possible when building a new feature, it reduces the points of maintenance.

What you are proposing:
```
local_bucket_hasher───────┐
                          │
                          │
s3Event_source────────────┤
                          │
                          │
submissions_api ──┬───────┴─────── pdq_hasher
                  │
                  │
                  ├─────────────── alternate_hasher
                  │
                  └─────────────── alternate_content_type
```

What it could be, thus benefiting automatically from new hashers:
```
local_s3_bucket│     ┼
               │
               │
api_calls──────┴──────────submissions_api ──┬─────────────── pdq_hasher
                                            │
                                            │
                                            ├─────────────── alternate_hasher
                                            │
                                            └─────────────── alternate_content_type
```